src/sentry/models/integrations/integration.py

```diff
@@ -106,18 +106,23 @@ def outboxes_for_update(identifier: int) -> List[ControlOutbox]:
             for region_name in find_regions_for_orgs(org_ids)
         ]
 
-    def add_organization(self, organization: RpcOrganization, user=None, default_auth_id=None):
+    def add_organization(
+        self, organization_id: int | RpcOrganization, user=None, default_auth_id=None
```

Comment:
nit: I think it'd be cleaner to just always have the caller supply the organization ID instead of passing in the full object if possible.